src/docfx/SubCommands/DocumentBuilderWrapper.cs

```diff
@@ -92,16 +93,24 @@ public static void BuildDocument(BuildJsonConfig config, TemplateManager templat
         {
             using (var builder = new DocumentBuilder(LoadPluginAssemblies(pluginDirectory)))
             {
-                var parameters = ConfigToParameter(config, templateManager, baseDirectory, outputDirectory);
-                if (parameters.Files.Count == 0)
-                {
-                    Logger.LogWarning("No files found, nothing is to be generated");
-                    return;
-                }
-
                 using (new PerformanceScope("building documents", LogLevel.Info))
                 {
-                    builder.Build(parameters);
+                    foreach (var parameters in ConfigToParameter(config, templateManager, baseDirectory, outputDirectory))
+                    {
```

Comment:
question: seems build per version, so currently we still couldn't handle cross version reference case?